src/EFCore.PG/Extensions/NpgsqlRangeExtensions.cs

```diff
@@ -32,6 +33,7 @@ namespace Microsoft.EntityFrameworkCore
     /// <summary>
     /// Provides extension methods for <see cref="NpgsqlRange{T}"/> supporting PostgreSQL translation.
     /// </summary>
+    [PublicAPI]
```

Comment:
We don't really use this Jetbrains annotation, it's truly you'll find it here and there but I'd rather avoid having it on every actual public method etc.